File: websocket-demo2/src/main/java/com/example/websocketdemo2/SeaBattle/SeaBattleMessage.java
package com.example.websocketdemo2.SeaBattle;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class SeaBattleMessage {
    private int status;
    private String player1;
    private String player2;
    private int x;
    private int y;
    private int hitStatus; // 0 - error, 1 - don't hit, 2 - hit
    private int shipId;
    private boolean destroyedShip; //
    private String turn;
}

File: WebSocketAngular2/src/app/models/SeaBattleMessage.ts
export class SeaBattleMessage {
  status!: number;
  player1!: string;
  player2!: string;
  x!: number;
  y!: number;
  hitStatus!: number; // -1 - error, 0 - don't hit, 1 - hit
  shipId!: number
  destroyedShip!: boolean;
  turn!: string;

}
